pkg/messaging/rabbitmq/publisher.go

```diff
@@ -65,7 +68,7 @@ func (pub *publisher) Publish(topic string, msg messaging.Message) error {
 		immediate,
 		amqp.Publishing{
 			Headers:     amqp.Table{},
-			ContentType: "text/plain",
+			ContentType: "application/octet-stream",
```

Comment:
You can use a const here

Reply:
This is linked with RabbitMQ branch [here](https://github.com/mainflux/mainflux/pull/1570)